src/vs/editor/contrib/codelens/codelensController.ts

```diff
@@ -402,6 +405,66 @@ export class CodeLensContribution implements IEditorContribution {
 			}
 		});
 	}
+
+	public getLenses(): CodeLensWidget[] {
+		return this._lenses;
+	}
+}
+
+export class ShowLensesInCurrentLineCommand extends Command {
```

Comment:
Use [`EditorCommand`](https://github.com/microsoft/vscode/blob/fd487a7ed6388a465b72cb9091b1a470536b68ed/src/vs/editor/browser/editorExtensions.ts#L141) instead of the generic command. It will already have the correct editor.

Reply:
Nice! Changed to use it. Thanks @jrieken